proto/substrait/algebra.proto

```diff
@@ -339,6 +339,23 @@ message ExchangeRel {
     }
   }
 }
+message ExpandRel {
```

Comment:
This is what I see the relation doing (based on my cursory glance at the referenced spark PR).  Please feel to adjust.

```suggestion

// The expand relation generates multiple rows per input row (denoted by 
// aggregate_expressions) which are then aggregated by groupings.  No
// ordering is applied to the aggregated results.
message ExpandRel {
```

Reply:
Updated. Thanks for your review!